python/ZvvHbb13TeVmacros/SavePlot.C

```diff
@@ -1,6 +1,6 @@
 {
  TFile* nf = new TFile("newfile.root","recreate");
- TFile *_file0 = TFile::Open("../env/ZvvHighPt_V21_MET.root");
+ TFile *_file0 = TFile::Open("/afs/cern.ch/work/s/swang373/public/V21_MVAout/ZvvHighPt_V21_MET.root");
```

Comment:
same as above, you and @silviodonato should probably agree on a relative path convention whenever needed